staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2784,6 +2786,13 @@ type PodAffinityTerm struct {
 	// selected pods is running.
 	// Empty topologyKey is not allowed.
 	TopologyKey string `json:"topologyKey" protobuf:"bytes,3,opt,name=topologyKey"`
+	// A label query over the set of namespaces that the term applies to.
+	// The term is applied to the union of the namespaces selected by this field
+	// and the ones listed in the namespaces field.
+	// null selector and null or empty namespaces list means "this pod's namespace"
```

Comment:
because people are often confused, should we explicitly say that an empty selector (`{}`) matches all namespaces?

Reply:
sg, done.